apps/condo/domains/common/utils/task/helpers.js

```diff
@@ -82,14 +86,34 @@ const getStartDates = (dateRaw) => {
     const date = getMonthStart(dateRaw)
 
     return {
+        prevMonthStart: date.subtract('1', 'month').format(DATE_FORMAT),
         thisMonthStart: date.format(DATE_FORMAT),
         nextMonthStart: date.add('1', 'month').format(DATE_FORMAT),
     }
 }
 
+/**
+ *
+ * @param context
+ * @returns {function(*, *, *, *=, *=): *[]}
+ */
+const getLoadByChunks = async (context) =>
```

Comment:
this util is redundant and just complicate the logic.

Reply:
It's ok to use in the bin but it's no OK to move in here.

Reply:
fixed